packages/cli/src/supermassive.ts

```diff
@@ -88,6 +80,10 @@ export function supermassive(): Command {
       "--enum-migration-exceptions-json-file [enumMigrationExceptionsJsonFile]",
       "File containing array of enum names, which should remain typescript enums",
     )
+    .option(
+      "--context-sub-type-metadata-file [contextSubTypeMetadataFile]",
+      "Subtype metadata file",
```

Comment:
would be great to have a more detailed comment here, possibly with file format example